utils/load_economy.lua

```diff
@@ -86,6 +87,9 @@ function load_eco(plrno, tribe, folder)
    -- hq = (tribe.."_headquarters")
    -- plr:place_building(hq, map.player_slots[plrno].starting_field, false, true)
 
+   assert(plr.tribe.name == eco.details[1].tribe,
```

Comment:
BTW is it even possible to load economy for a non-existent player? If not, then we don't even need the `tribe` argument for `load_eco()`.